### DISCUSSION TITLE
CPU and Memory scaler

### DISCUSSION THREAD

**Discussion by visla-xugeng** [kedacore/keda#2784] [Q&A / Need Help] [answered]
Hi KEDA community,

The trigger type of CPU or Memory can be type: Utilization/ AverageValue.

How does KEDA to fetch the utilization or AverageValue of CPU? What I mean is that how KEDA sample the value at the point when the KEDA poll the pod? The value KEDA returned by poll is the average of multiple values during the polling interval or just the one value at particular point?

**Comment by zroubalik** [comment, marked as answer]
Hi, this scaler is "just a wrapper" around HPA and it's build in capabilities on CPU/Memory driven autoscaling. https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/

**Reply by visla-xugeng** [reply]
Thanks a lot. @zroubalik
